docs/source/introduction/description_geoips.rst

```diff
@@ -3,7 +3,6 @@
  | # # # This source code is protected under the license referenced at
  | # # # https://github.com/NRLMMD-GEOIPS.
 
-*********************
```

Comment:
https://devguide.python.org/documentation/markup/#sections

Are the overlines incorrect? I don't care either way,  just curious,  and we should pick a standard and stick with it in any case.

Reply:
Hm, it looks like there's inconsistant documentation about this - the documentation above says overlines are optional, some sphinx documentation requests `*`s overlines and no others, while other sections of the sphinx documentation request (specifically) no overlines. Right now pink removes overlines, so that's what's done here, but I don't have any attachment to having them/vs not.

Reply:
I agree we should probably standardize this! 🙂

Reply:
I don't have any attachment either way either!

Reply:
https://github.com/NRLMMD-GEOIPS/geoips/issues/720

Created an issue just for fun - probably will resolve it just by saying we will remove overlines, and stick with current pink formatting.